lms/templates/fields/field_image.underscore

```diff
@@ -0,0 +1,15 @@
+<div class="image-wrapper">
+    <img class="image-frame" src="<%- gettext(imageUrl) %>"  alt="<%=imageAltText%>"/>
+    <div class="u-field-actions">
+        <a class="u-field-upload-button">
```

Comment:
Can you make this a ```<label>``` element?  then the <%= uploadButtonTitle %> will serve as the label for the input